content/user/ui/report/Org-stats.md

```diff
@@ -0,0 +1,51 @@
+<!--
+title: "Organization Statistics"
+description: "Overview of organization statistics charts"
+tags: "TeamServer licenses applications servers organizations report"
+-->
+
+For a summary of administrative statistics on your Contrast installation, look no further than your Organization Statistics. 
```

Comment:
For some reason this doesn't sound like our article voice to me. Can we try rephrasing? Example: "Organization Statistics provides robust and comprehensive visibility into your most crucial organization data."